src/components/registerForm.js

```diff
@@ -22,50 +22,67 @@ class RegisterForm extends Component {
 
 	render() {
 		let { email, password } = this.state.form.user
+		console.log(this.auth.loggedIn())
 		return (
-			<main>
-				<h2>Welcome! Register here.</h2>
-				<form className="Formfield" onSubmit={this.onSubmit}>
-					<input
-						type="email"
-						name="email"
-						value={email}
-						onChange={this.onChange}
-					/>
-					{this.state.errors.email && <div>Error: Email  {this.state.errors.email[0]}</div>}
+			<div>
+				<Grid>
```

Comment:
Tabs versus spaces ?  Very nit-picky - PR is still approved.   Just a note.  Be consistent across your project.